service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -311,7 +313,7 @@ func (t *transferQueueActiveTaskExecutor) processCloseExecution(
 	if replyToParentWorkflow {
 		ctx, cancel := context.WithTimeout(context.Background(), transferActiveTaskDefaultTimeout)
 		defer cancel()
-		_, err = t.historyClient.RecordChildExecutionCompleted(ctx, &historyservice.RecordChildExecutionCompletedRequest{
+		_, err := t.historyClient.RecordChildExecutionCompleted(ctx, &historyservice.RecordChildExecutionCompletedRequest{
```

Comment:
This will shadow `err` declared above.  Is this on purpose?  I don't get it why creating a new `err` in this scope matters.

Reply:
there is no behavior change
the intention here is to check the error and return if seeing error